test/spec/modules/33acrossAnalyticsAdapter_spec.js

```diff
@@ -438,17 +464,23 @@ function mapToBids(auctions) {
 
 function getLocalAssert() {
   function isValidAnalyticsReport(report) {
-    assert.containsAllKeys(report, ['analyticsVersion', 'pid', 'src', 'pbjsVersion', 'auctions']);
-    if ('usPrivacy' in report) {
-      assert.match(report.usPrivacy, /[0|1][Y|N|-]{3}/);
+    try {
+      assert.containsAllKeys(report, ['analyticsVersion', 'pid', 'src', 'pbjsVersion', 'auctions']);
+      if ('usPrivacy' in report) {
+        assert.match(report.usPrivacy, /[0|1][Y|N|-]{3}/);
+      }
+      assert.equal(report.analyticsVersion, '1.0.0');
+      assert.isString(report.pid);
+      assert.isString(report.src);
+      assert.equal(report.pbjsVersion, '$prebid.version$');
+      assert.isArray(report.auctions);
+      assert.isAbove(report.auctions.length, 0);
+      report.auctions.forEach(isValidAuction);
+    } catch (e) {
```

Comment:
is the try/catch statement really necessary? I think no malformed mocked report gets sent. I am not seeing those lines being executed.

Reply:
My recommendation is to remove it since it's no longer needed, just to avoid any confusion about an existing spec that we would expect to execute that catch code.